web/crux-ui/src/components/projects/versions/deployments/deployment-view-list.tsx

```diff
@@ -57,9 +57,16 @@ const DeploymentViewList = (props: DeploymentViewListProps) => {
     <span suppressHydrationWarning>
       {item.image.createdAt ? utcDateToLocale(item.image.createdAt) : t('common:new')}
     </span>,
-    <Link href={routes.deployment.instanceDetails(state.deployment.id, item.id)} passHref>
-      <DyoIcon src="/settings.svg" alt={t('common:settings')} size="md" />
-    </Link>,
+    <div>
+      <div className="inline-block mr-2">
+        <Link href={routes.project.versions(state.project.id).imageDetails(state.version.id, item.image.id)} passHref>
+          <DyoIcon src="/image_config_icon.svg" alt={t('common:settings')} size="md" />
```

Comment:
Both settings button have "Settings" as their tooltip, isn't it a little confusing?